### PR TITLE
Carthage support

### PR DESCRIPTION
**Pull request by prettysunflower**

### CODE REVIEW

**Comment by ldiqual** [comment]
@RemyDCF Thanks a lot for this! Can you describe what you actually did here?

**Comment by prettysunflower** [comment]
I marked the SwiftyButton scheme as shared for adding the Carthage support (as described at https://github.com/Carthage/Carthage). I also updated the README.md.

File: README.md [hunk 1]
```diff
@@ -1,17 +1,29 @@
-SwiftyButton
+SwiftyButton
```
Comment (ldiqual) [inline README.md]: Can you discard those changes, it clutter the diff a bit

**Review by ldiqual — CHANGES_REQUESTED** [review]
@RemyDCF Awesome, thanks for the explanations. Just one minor fix and it'll be good to go. Can you also add something in the changelog? Thank you

**Comment by prettysunflower** [comment]
Which version number I add to the Changelog ?

File: README.md (hunk 1 above)
Comment (prettysunflower) [reply README.md]: Done

**Comment by ldiqual** [comment]
@RemyDCF Add it in the `master` section.

**Comment by prettysunflower** [comment]
Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ Change Log
  - **Breaking**: SwiftyButton has been split into two different button types: `FlatButton` and `PressableButton`. Please refer to the [README](./README.md) for more details.
  - Setting `cornerRadius` on a `CustomPressableButton` now adjusts the corner radius of the content view.
    [#4](https://github.com/TakeScoop/SwiftyButton/issues/4)
+ - Adding Carthage support.
```
Comment (ldiqual) [inline CHANGELOG.md]: This is in the `0.6.0` section, not the master section.

**Review by ldiqual — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (prettysunflower) [reply CHANGELOG.md]: Ok, I changed this.

**Comment by ldiqual** [comment]
Perfect, thank you so much for your contribution!

**Pull request merged**